lib/components/Provider/index.tsx

```diff
@@ -24,6 +26,8 @@ import {
 export type Context = {
   notifications?: InAppNotification[];
   preferences?: GetPreferencesResponse;
+  userAccountMetaData?: { userAccountMetadata: UserAccountMetadata };
+  hideWebPushOptInMessage?: boolean;
```

Comment:
webPushOptInMessage: "AUTOMATIC" | boolean
// automatic -> show when web push is enabled
// true -> show even if there is no web push enabled
// false -> never show